src/fights.ts

```diff
@@ -1535,13 +1538,44 @@ const freeRunFightSources = [
     () =>
       (get("gingerbreadCityAvailable") || get("_gingerbreadCityToday")) &&
       get("_gingerbreadCityTurns") + (get("_gingerbreadClockAdvanced") ? 5 : 0) === 19 &&
-      availableAmount($item`sprinkles`) > 5,
+      (availableAmount($item`sprinkles`) > 5 || haveOutfit(`gingerbread best`)),
     () => {
       propertyManager.setChoices({
         1203: 4, // Gingerbread Civic Center 5 gingerbread cigarettes
         1215: 1, // Gingerbread Civic Center advance clock
+        1209: 2, // enter the gallery at Upscale Midnight
+        1214: 1, // get High-End ginger wine
       });
-      adventureMacro($location`Gingerbread Civic Center`, Macro.abort());
+      const boozes = $items`elemental caipiroska, moreltini, Dreadsylvanian grimlet, Hodgman's blanket, Sacramento wine, iced plum wine, splendid martini, Eye and a Twist, jar of fermented pickle juice, dirt julep, Ambitious Turkey, Friendly Turkey`;
```

Comment:
i would prefer if this list was somehow generated from menu() and potionMenu() instead of being hard coded here. Possibly this whole next block of code could be a function in diet.ts